src/features/content-explorer/content-explorer/__tests__/ContentExplorer.test.js

```diff
@@ -106,6 +106,113 @@ describe('features/content-explorer/content-explorer/ContentExplorer', () => {
             expect(wrapper.find('ContentExplorerSelectAll').prop('isSelectAllChecked')).toEqual(isSelectAllChecked);
         });
 
+        test('should render ContentExplorerIncludeSubfolders with canIncludeSubfolders = true and isSelectAllAllowed = true', () => {
+            const isSelectAllAllowed = true;
+            const canIncludeSubfolders = true;
+            const wrapper = renderComponent({ canIncludeSubfolders, isSelectAllAllowed });
+
+            expect(wrapper.exists('ContentExplorerIncludeSubfolders')).toBeTruthy();
+        });
+
+        test('should not render ContentExplorerIncludeSubfolders with canIncludeSubfolders = false and isSelectAllAllowed = true', () => {
+            const isSelectAllAllowed = true;
+            const canIncludeSubfolders = false;
+            const wrapper = renderComponent({ canIncludeSubfolders, isSelectAllAllowed });
+
+            expect(wrapper.exists('ContentExplorerSelectAll')).toBeTruthy();
+            expect(wrapper.find('ContentExplorerIncludeSubfolders').length).toBe(0);
```

Comment:
you're using both to `exists` and `find`. you should be able to do:
`expect(wrapper.exists('ContentExplorerIncludeSubfolders')).toBe(false);`